composer.json

```diff
@@ -11,36 +11,36 @@
         }
     ],
     "require": {
-        "php": "^7.2 || ^8.0",
+        "php": "^7.3 || ^8.0",
```

Comment:
conflict with prophecy require reach the PHP version like we did for the task packages already